project/controllers/user_cntr.py

```diff
@@ -0,0 +1,15 @@
+from ..models import User, users
+
+
+class UserController:
+
+    def create_user(self, user):
+        new_user = user.user_to_json()
+        users.append(new_user)
+        return users
+
+    def get_all_users(self):
+        for user in users:
```

Comment:
In this function just return the entire users list
```suggestion
        return users
```